packages/frontend/src/pages/AdminLogin.tsx

```
@@ -137,247 +109,86 @@
+
+            <form
+              onSubmit={(e) => {
+                e.preventDefault();
+                e.stopPropagation();
+                void form.handleSubmit();
+              }}
+            >
+              <div className="flex flex-col gap-6">
+                <form.Field
+                  name="password"
+                  children={(field) => (
```

Biome (2.1.2)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)